package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "knowledge-explorer",
+  "version": "1.0.0",
+  "description": "Knowledge Explorer project for Hacktoberfest conducted by Devopedia.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ikmrgrv/knowledge-explorer.git"
```

Comment:
Update URL to point to DevopediaOrg repo. Update other URLs in this file.

Reply:
have updated it.